src/AspNet.Security.OAuth.Untappd/UntappdAuthenticationHandler.cs

```diff
@@ -29,6 +33,63 @@ public class UntappdAuthenticationHandler : OAuthHandler<UntappdAuthenticationOp
         {
         }
 
+        protected override async Task<OAuthTokenResponse> ExchangeCodeAsync([NotNull] OAuthCodeExchangeContext context)
+        {
+            var tokenRequestParameters = new Dictionary<string, string>()
+            {
+                { "client_id", Options.ClientId },
+                { "redirect_uri", context.RedirectUri },
+                { "client_secret", Options.ClientSecret },
+                { "code", context.Code },
+                { "grant_type", "authorization_code" },
```

Comment:
Newer initializer syntax is preferred.

```suggestion
                ["client_id"] = Options.ClientId,
                ["redirect_uri"] = context.RedirectUri,
                ["client_secret"] = Options.ClientSecret,
                ["code"] = context.Code,
                ["grant_type"] = "authorization_code",
```